CODEOWNERS

```diff
@@ -62,6 +62,7 @@
 /soc/arm/renesas_rcar/                    @aaillet @pmarzin
```

Comment:
do you get this:
![image](https://user-images.githubusercontent.com/74421316/206441954-da142bc9-af8a-47f1-bf1c-19f652a88a9c.png)
?

You should ask for permissions on zephyr. 

David Leach can help you with this.

Reply:
Yup, it also seems to be the case for the initial PR that introduced the qemu_cortex_a53 ([1]) so I didn't pay much attention to it.

[1]: https://github.com/zephyrproject-rtos/zephyr/pull/20263/commits/f25e737be4f4d4cecf7ef93eb4e6d1d003146767#diff-e665c2ec603df9ae1f15d1131e5feb9dd949bde111b958777e7f997fcd5ee52f

Reply:
ok, got it!
But you should still be added as developer in zephyr, to be able to create PRs, add reviewers & other stuff